docker/postgis/Dockerfile

```
@@ -36,6 +36,16 @@ RUN set -eux  ;\
         libkakasi2-dev \
         libgdal-dev \
     ;\
+    ## GEOS
+    cd /opt/ ;\
+    curl -o /opt/geos.tar.bz2 http://download.osgeo.org/geos/geos-3.9.1.tar.bz2 ;\
+    mkdir /opt/geos ;\
+    tar xf /opt/geos.tar.bz2 -C /opt/geos --strip-components=1 ;\
+    cd /opt/geos/ ;\
+    ./configure ;\
+    make -j ;\
+    make install ;\
+    rm -rf /opt/geos* ;\
```

please add this line before line 10:

```
ARG GEOS_VER=3.9.1
```

and add an extra space for consistency with other code (plus use the above var):

```suggestion
    ## GEOS
    cd /opt/  ;\
    curl -o /opt/geos.tar.bz2 http://download.osgeo.org/geos/geos-${GEOS_VER}.tar.bz2  ;\
    mkdir /opt/geos  ;\
    tar xf /opt/geos.tar.bz2 -C /opt/geos --strip-components=1  ;\
    cd /opt/geos/  ;\
    ./configure  ;\
    make -j  ;\
    make install  ;\
    rm -rf /opt/geos*  ;\
```

---

Done. Thanks.